packages/broker/test/integration/plugins/operator/OperatorPlugin.test.ts

```diff
@@ -23,7 +23,7 @@ describe('OperatorPlugin', () => {
             privateKey: brokerWallet.privateKey,
             extraPlugins: {
                 operator: {
-                    operatorContractAddress: '0x4A5C0EC07F7ddBd4B6050181638e24b0153991b2'
+                    operatorContractAddress: '0x139dfa493a45364b598f2f98e504192819082c85'
```

Comment:
Do we have a private key for this address? If we could calculate the address from it?

Reply:
its an operator contracts address, not a wallet